db/migrate/20200530162659_add_status_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToOrders < ActiveRecord::Migration[5.1]
```

Comment:
Added status to orders table using enums. Not 100% sure how it will work, but default is 0 which equal "pending". Please take note if your methods need to retrieve or update the order status!

Reply:
Woot woot way to go, I expected that you were just going to use strings, way to go the extra step.